front/src/pods/hotel-edit/hotel-edit.validation.spec.ts

```diff
@@ -0,0 +1,238 @@
+import { FieldValidationResult } from 'lc-form-validation';
+import { HotelEditFormValidation } from './hotel-edit.validation';
+
+describe('Hotel Edit Validation tests', () => {
+    describe('Field Validations =>', () => {
+        it('should invalidate when hotel name does not exist', (done) => {
+
+            // Arrange
+            const viewModel = {
+                name: 'arts',
+                city: 'Barcelona',
+                description: 'Más de 10 letras para que pete'
+            };
+
+            // Act
+            HotelEditFormValidation
+                .validateField(viewModel, 'name', "")
+                .then(fieldValidationResult => {
+                    // assert
+                    expect(fieldValidationResult).toEqual({
+                        errorMessage: 'Please fill in this mandatory field.',
+                        key: 'name',
+                        succeeded: false,
+                        type: 'REQUIRED'
+                    } as FieldValidationResult);
```

Comment:
In this implementation you are relying on message value as well, it can be ok, you can only check type and succeeded values that could be another option